src/cli/README.md

```diff
@@ -868,6 +869,35 @@ Note: Only for certification test.
 Done
 ```
 
+### fem
+
+Get external FEM parameters.
+
+```bash
+> fem lnagain
+LNA gain 11 dBm
+Done
+```
+
```

Comment:
Is there a use case for this? Are we planning to have other FEM parameters here?

If we want to keep this, is this covered by the expect tests?

Reply:
Per #5561, there are needs to set FEM gain to radio so that when setting/getting CCA threshold, user does not need to offset the LNA gain anymore. There might be other FEM parameters added later, like PA gain to cap the tx power.

This cli is covered in expect tests in cli-misc.exp

Reply:
I see `fem lnagain` is covered, but not `fem`.

Note that the doc above seems incorrect for the example output. It should be `fem` with zero args, correct?

Reply:
Sorry, my mistake. Updated.